### PR TITLE
Add KrazyKleb Domain

### PR DESCRIPTION
**Pull request by KrazyKleb999**
For my "Portfolio"

### Link to Website

Link: portfoilio-production-2a0f.up.railway.app

---

- [ ] Toggle this checkbox to re-run the stargazer checking

### CODE REVIEW

File: domains/KrazyKleb.is-an.app.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+	"$schema": "../schemas/domain.schema.json",
+	"description": "KrazyKleb's website",
+	"domain": "is-an.app",
+	"subdomain": "KrazyKleb",
+	"owner": {
```
Comment (tarampampam) [inline domains/KrazyKleb.is-an.app.json:6]: In lower case, please

File: domains/KrazyKleb.is-an.app.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (tarampampam) [inline domains/KrazyKleb.is-an.app.json:1]: File name in lower case, please

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: domains/KrazyKleb.is-an.app.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+	"$schema": "../schemas/domain.schema.json",
```
Comment (tarampampam) [inline domains/KrazyKleb.is-an.app.json:2]: indent size - 2 spaces (everywhere)

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: domains/KrazyKleb.is-an.app.json (hunk 1 above)
Comment (KrazyKleb999) [reply domains/KrazyKleb.is-an.app.json]: Done!

File: domains/KrazyKleb.is-an.app.json (hunk 2 above)
Comment (KrazyKleb999) [reply domains/KrazyKleb.is-an.app.json:1]: Done!

Comment (tarampampam) [reply domains/KrazyKleb.is-an.app.json:1]: Nope :(

**Comment by tarampampam** [comment]
closed due inactivity

**Pull request closed without merging**